src/Tribe/Aggregator/Migrate.php

```diff
@@ -245,6 +245,30 @@ public function is_ical_migrated() {
 	}
 
 	/**
+	 * Filters out empty values
+	 *
+	 * NOTE: we aren't using array_filter because EVEN with an empty() alias, the results are
+	 * unpredictable
+	 *
+	 * @param array $original_values
+	 *
+	 * @return array
+	 */
+	public function filter_out_unwanted_values( $original_values ) {
```

Comment:
Purely for my own ref I'd love to know more about when and where array_filter was letting us down here. That aside, as a general purpose helper that could be useful in all kinds of places, might it be better housed someplace inside tribe-common?